### PR TITLE
Add acceptance criteria to bug, task, and redesign task

### PR DESCRIPTION
**Pull request by agarzola**
## Description

This PR adds an _Acceptance Criteria_ section to our Bug, Task, and Redesign Task issue templates.

## Motivation / Context

Proposed during a recent Monthly Front-end Council meeting and approved by @lina-calin.

## Testing Instructions / How This Has Been Tested

A proof-read should suffice!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -43,6 +43,14 @@ assignees: ''
 
 <!-- A description of what you expected to happen. -->
 
+## Acceptance Criteria
+
+<!-- A list of one or more requirements that must be met in order to for -->
+<!-- this ticket to be considered “done.” Use checkboxes to mark each item -->
+<!-- as an actionable task. -->
+
+- [ ] tk…
```
Comment (danarbello) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
- [ ] TK
```

This is so stupid and nitpicky that I wouldn't dare have it hold up this ticket, but two things here 🤦 I'm sorry…
- Can it be one word to make it one key arrow tap shorter when selecting and replacing the "TK"? (Blame yourself for showing me this key command in the first place 😏)
- If you're doing that, then you might as well update it to be a weeeeee bit less ambiguous with a classic "TK" (all uppercase like heyday editorial redlining) 🤷 

_If_ you let me be that kind of asshole, then you'd update the other two templates, too.

**Review by danarbello — APPROVED** [review]
💥

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -43,6 +43,14 @@ assignees: ''
 
 <!-- A description of what you expected to happen. -->
 
+## Acceptance Criteria
+
+<!-- A list of one or more requirements that must be met in order to for -->
```
Comment (danarbello) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
<!-- A list of one or more requirements that must be met in order for -->
```

Same for the other two templates.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (agarzola) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is the kind of nitpicky assholery that I’m here for! 😍 I’ll commit this suggestion now. Thank you!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (agarzola) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Fixed on all three and re-wrapped the comment in 8776ca5. Thank you!

**Comment by lina-calin** [comment]
This looks great to me! We might want to keep an eye out in the future if formatting recommendations would be helpful (like is done with user stories; "as a __ I want to ___") but barring that, this is fantastic!!! Thank you, Alfonso!

**Review by lina-calin — APPROVED** [review]

**Pull request merged**